FirebasePerformance/Sources/AppActivity/FPRAppActivityTracker.h

```diff
@@ -50,6 +51,9 @@ NS_EXTENSION_UNAVAILABLE("Firebase Performance is not supported for extensions."
 /** Current running state of the application. */
 @property(nonatomic, readonly) FPRApplicationState applicationState;
 
+/** Firebase Remote Configuration object */
```

Comment:
This is Firebase Performance configuration object. Remote Config configuration object is abstracted within.